src/views/Affiliates/LastUpdated.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+// Styled component for the container
+const Container = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  font-family: 'Inter', sans-serif;
+`;
+
+interface IProps {
+  lastUpdatedDate: Date;
+}
+
+const LastUpdated = ({ lastUpdatedDate }: IProps) => {
+  const stringGetter = useStringGetter();
+
+  // State to hold the current time
+  const [currentTime, setCurrentTime] = useState<Date>(new Date());
+
+  // Update the current time every minute
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setCurrentTime(new Date());
+    }, 60000); // Update every 60 seconds
```

Comment:
nit: can move numbers into a constant at the time of the file

Reply:
Not sure on what you mean here. The last updated is backend based. It is the last time the affiliates server synced with the dydx trading data.